clients/ui/frontend/package.json

```diff
@@ -29,18 +28,22 @@
     "cypress:server": "serve ./dist -p 9001 -s -L"
   },
   "devDependencies": {
+    "@babel/preset-env": "^7.21.5",
+    "@babel/preset-react": "^7.18.6",
+    "@babel/preset-typescript": "^7.21.5",
     "@cypress/code-coverage": "^3.12.45",
     "@testing-library/cypress": "^10.0.1",
-    "@testing-library/jest-dom": "^6.4.8",
-    "@testing-library/react": "^16.0.0",
+    "@testing-library/jest-dom": "^6.3.0",
+    "@testing-library/react": "^14.0.0",
     "@testing-library/user-event": "14.4.3",
-    "@types/jest": "^29.5.12",
+    "@types/jest": "^28.1.8",
```

Comment:
@Griffin-Sullivan do we need to downgrade those versions? Can we start with the most recent ones?